docs/source/online/sharingExperiments.rst

```diff
@@ -0,0 +1,81 @@
+.. include:: ../global.rst
+
+.. _sharingExperiments:
+
+Pavlovia contains a collection of experiments that you can use as a starting point for your own experiment. Below we explain how you can search for experiments in this collection and contribute your own experiment.
+
+Searching for experiments on Pavlovia
+-------------------------------------
+You can search for experiments via the Pavlovia website and from within the PsychoPy Builder.
+
+Via the Pavlovia website
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the `Pavlovia`_ home page, you can explore your own existing projects, or other users' public projects. To find a project, go to `Pavlovia's Explore page <https://pavlovia.org/explore>`_ (see Figure 1).
+
+.. figure:: /images/pavlovHome.png
+    :align: center
+    :alt: alternate text
+    :figclass: align-center
+
+*Figure 1*. The `Pavlovia`_ home page
```

Comment:
See note about using figure captions as part of the figure and I'd use the automatic figure numbering too

Reply:
Done :)